content/js/query.md

```diff
@@ -76,40 +61,36 @@ skygear.publicDB.query(query).then((notes) => {
 });
 ```
 
-### Counting the records
+<a name="conditions"></a>
+## Conditions
 
-To get the number of records matching a query, set the `overallCount`
-of the Query to `true`. In this case, you will get the count of all records
-matching the query together with the query result.
+Here is a list of simple conditions you can apply on a query:
 
 ``` javascript
-let query = new Query(Note);
-query.lessThan('order', 10);
-query.overallCount = true;
-skygear.privateDB.query(query).then((notes, count) => {
-  console.log('%d records matching query.', count);
-}, (error) => {
-  console.log('error: ', error);
-});
+query.equalTo('title', 'First note');      // ===
+query.notEqualTo('category', 'dangerous'); // !==
+query.lessThan('age', 10);                 // <
+query.lessThanOrEqualTo('age', 10);        // <=
+query.greaterThan('size', 50);             // >
+query.greaterThanOrEqualTo('size', 50);    // >=
 ```
 
-The count is not affected by the limit set on the query. So, if you only want
-to get the count without fetching any records, simple set `query.limit = 0`.
-
 ### Contains condition
 
-The `contains` function can be used to query a key for value that matches one of the
-item in a specified array.
+The `contains` condition can be used to query a key for value that matches one
+of the items in a specified array.
 
 ```javascript
 query.contains('category', ['interesting', 'funny']);
+query.notContains(...);
 ```
 
-If the value is an array, the `containsValue` function can be used to query for
+If the value is an array, the `containsValue` condition can be used to query for
```

Comment:
seems like the distinction for `contains` and `containsValue` is not expressed clearly. perhaps you can rephrase them, or with more concrete examples?